checkov/common/output/record.py

```diff
@@ -228,7 +228,7 @@ def get_output_id(self, use_bc_ids: bool) -> str:
         return self.bc_check_id if self.bc_check_id and use_bc_ids else self.check_id
 
     def get_unique_string(self) -> str:
-        return f"{self.check_id}.{self.check_result}.{self.file_abs_path}.{self.file_line_range}.{self.resource}"
+        return f"{self.check_id}.{self.file_abs_path}.{self.file_line_range}.{self.resource}"
```

Comment:
why is this changed?

Reply:
This is the solution for filtering out checks with same resource id. Until we tackle it in a broader way this seems as the simplest and fastest approach that won't potentially break things. It seems we could've used this unique string all along and there was no real need for the check result to be part of this string.

Reply:
So 2 things here:
1. why does this even solve the uniqueness? In the case we saw togther the check_result was the same, so it seems like this won't be the right thing to solve (perhaps it wasn't the case here but we could have 2 checks with the same results on the same resource, so this won't filter it correctly).
2. I would like some way of knowing that the change we do here is temporary. For example - adding another function with clear documentation about why do we handle/remove duplicates. This way it will be better documented for others as well

Reply:
Discussed offline,
After the resource_id eventual fix we won't need to revert this since it's unique enough